docs/contributing/documentation.rst

```diff
@@ -20,27 +20,37 @@ your work.
 
 Steps for contributing documentation:
 
+-  Follow the workflow for :ref:`contributing` to Anemoi, including
+   creating an issue and submitting a Pull Request.
+
 -  Identify the most appropriate place for your addition (e.g., overall
    `Anemoi documentation <https://github.com/ecmwf/anemoi-docs>`_,
    package-level documentation), and the relevant section (e.g., user
    guide, getting started, or developer guide).
 
--  Consider maintainability — are interfaces still evolving?
-
--  Write clear and concise documentation using simple, direct sentences.
+-  You find the documentation in the `docs` directory of the Anemoi
+   repository. The documentation is written in reStructuredText (reST)
+   format. Add your changes to the appropriate file in the `docs`
+   directory and pay attention to the following:
 
--  Add references to related sections or external documentation where
-   applicable.
+   -  Consider maintainability — are interfaces still evolving?
+   -  Write clear and concise documentation using simple, direct
+      sentences.
+   -  Add references to related sections or external documentation where
+      applicable.
 
 -  Incorporate feedback from your reviewer regarding documentation
    clarity and completeness.
 
+If you are unsure where to add documentation or how to structure it,
+please feel free to ask for help on the issue you opened at any point.
+
 Building Documentation
 ======================
 
-For each Pull Request, documentation is automatically built and hosted
-on ReadTheDocs for review. It is automatically linked in the PR
-description.
+For each Pull Request that modifies the files in the `docs` folder,
+documentation is automatically built and hosted on ReadTheDocs for
+review. It is automatically linked in the PR description.
 
```

Comment:
I would mention here, that is the contributor responsibility to check that the auto generated links are build correctly and can be visualised? If the links are failing they should check the CI workflow associated to [readthedocs-pr-update.yml](https://github.com/ecmwf/anemoi-core/blob/main/.github/workflows/readthedocs-pr-update.yml) or the logs from the build action for the relevant project like (https://app.readthedocs.org/projects/anemoi-training/builds/27886217/). If in doubt they can check with an anemoi maintainer